packages/contract-tests/test/errors.js

```diff
@@ -83,34 +87,26 @@ describe("Client appends errors (vmErrorsOnRPCResponse)", function () {
       }
     });
 
-    it("should append original stacktrace for OOG errors", async function () {
+    it("appends original stacktrace for OOG errors", async function () {
       try {
         await Example.new(1, { gas: 10 });
         assert.fail();
       } catch (e) {
         assert(
-          e.stack.includes("Error: base fee exceeds gas limit"),
+          e.stack.includes("Error: intrinsic gas too low"),
           "Should keep hijacked error description"
         );
         assert(
           e.stack.includes("/test/errors.js:"),
           "Should include original stack details"
         );
-        assert(
```

Comment:
To be honest I don't really know why these were here in the first place but after discussion it seemed like something we didn't care about (maybe we did at one point). I probably should remove the other ones as well, these were just the ones that caused tests to fail.

Reply:
I removed a couple of assertions involving directly checking `e.hijackedStack`. I left in the tests whose descriptions reference hijacked stack because some of them appear like they could still be relevant.

Reply:
(The other ones have also been removed)

Reply:
OK, seems these have been removed now.